docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+services:
+  app:
+    build: .
+    env_file: .env
```

Comment:
I'm honestly not a fan of using `env_file` here, because docker compose won't start up unless that's present. The app itself can use the `dotenv` file to load those env vars right?

Reply:
Yes, the project can use `dotenv` . This is resolved in commit [5177b1c](https://github.com/mattermost/mattermost-app-google-drive/pull/2/commits/5177b1c4f207b98ae69a429ce4b953c1557373ce)